crates/biome_service/src/file_handlers/mod.rs

```diff
@@ -731,6 +732,272 @@ impl<'a> RegistryVisitor<GraphqlLanguage> for SyntaxVisitor<'a> {
     }
 }
 
+/// Type meant to register all the lint rules for each language supported by Biome
+///
+#[derive(Debug)]
+pub(crate) struct LintVisitor<'a, 'b> {
+    pub(crate) enabled_rules: Vec<RuleFilter<'a>>,
+    pub(crate) disabled_rules: Vec<RuleFilter<'a>>,
+    lint_params: &'b LintParams<'a>,
```

Comment:
In #3496, I added `only` and `skip` on `FixAllParams`. We will need to avoid the dependency on `LintParams`.

Reply:
That's done in [`036b1c8` (#3483)](https://github.com/biomejs/biome/pull/3483/commits/036b1c84e202602f8610fb30b56a59ff699f71c1)